src/components/HypothesisTesting/SimulateTypeOneError.js

```diff
@@ -87,12 +88,13 @@ export default function SimulateTypeOneError({ popShape, mu0, alpha, distType, s
 
         const sampleObject = {
           testStatistic: _.round(testStatistic, 2),
-          mean: _.round(sampleMean, 2),
+          meansDiff: _.round(sampleMean2 - sampleMean, 2),
+          mean: testType === "oneSample" ? _.round(sampleMean, 2) : meansDiff,
```

Comment:
You could even combine lines 91 and 92: `mean: testType === "oneSample" ? _.round(sampleMean, 2) : _.round(sampleMean2 - sampleMean, 2)` since we're not using the `meansDiff` property anywhere.

Reply:
Also, I believe this should be `sampleMean - sampleMean2`